packages/uploadthing/src/client.ts

```diff
@@ -90,9 +103,10 @@ export const DANGEROUS__uploadFiles = async <TRouter extends FileRouter>(
 ) => {
   // Get presigned URL for S3 upload
   const s3ConnectionRes = await fetch(
-    createRequestPermsUrl({
+    createRequestUrl({
```

Comment:
nit: might be worth naming this "createS3RequestUrl" or something to make it clear what it's calling specifically